beacon-chain/blockchain/receive_attestation.go

```diff
@@ -33,13 +33,18 @@ func (s *Service) ReceiveAttestationNoPubsub(ctx context.Context, att *ethpb.Att
 	defer span.End()
 
 	// Update forkchoice store for the new attestation
-	if err := s.forkChoiceStore.OnAttestation(ctx, att); err != nil {
+	indices, err := s.forkChoiceStoreOld.OnAttestation(ctx, att)
```

Comment:
so we're still running both fork choices at the same time?

Reply:
I don't want to deprecate the old one until this has been battle tested. Either fork choice can be switch over via `--proto-array-forkchoice`